src/Microsoft.IdentityModel.Tokens/Microsoft.IdentityModel.Tokens.csproj

```diff
@@ -62,6 +62,11 @@
     <PackageReference Include="System.Threading" Version="4.3.0-*" />
     <PackageReference Include="System.Xml.XmlDocument" Version="4.3.0" />
   </ItemGroup>
+  
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0' ">
+    <PackageReference Include="Microsoft.CSharp" Version="4.5.0" />
```

Comment:
Not really sure why this package reference is needed. Could you please explain?

Reply:
Are you using dynamic anywhere?

Reply:
Oh, got it. Thank you.

Reply:
You should just remove the use of dynamic though.